### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by vongohren** [AMDmi3/jsonslicer#39]
Im quite new to Python and run on a mac.
I just wonder how I can use this lib?

Im used to be able to pip install, and then i got this error.

```
Collecting jsonslicer
  Downloading jsonslicer-0.1.5.tar.gz (23 kB)
    ERROR: Command errored out with exit status 1:
     command: /Users/snorre/.pyenv/versions/3.6.7/bin/python3.6 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/50/k8zgk6715jngcvgt2ryh99xm0000gn/T/pip-install-fyo2h4xw/jsonslicer/setup.py'"'"'; __file__='"'"'/private/var/folders/50/k8zgk6715jngcvgt2ryh99xm0000gn/T/pip-install-fyo2h4xw/jsonslicer/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/50/k8zgk6715jngcvgt2ryh99xm0000gn/T/pip-install-fyo2h4xw/jsonslicer/pip-egg-info
         cwd: /private/var/folders/50/k8zgk6715jngcvgt2ryh99xm0000gn/T/pip-install-fyo2h4xw/jsonslicer/
    Complete output (15 lines):
    Package yajl was not found in the pkg-config search path.
    Perhaps you should add the directory containing `yajl.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'yajl' found
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/50/k8zgk6715jngcvgt2ryh99xm0000gn/T/pip-install-fyo2h4xw/jsonslicer/setup.py", line 87, in <module>
        **pkgconfig('yajl')
      File "/private/var/folders/50/k8zgk6715jngcvgt2ryh99xm0000gn/T/pip-install-fyo2h4xw/jsonslicer/setup.py", line 16, in pkgconfig
        for token in subprocess.check_output(['pkg-config', '--libs', '--cflags', package]).decode('utf-8').split():
      File "/Users/snorre/.pyenv/versions/3.6.7/lib/python3.6/subprocess.py", line 336, in check_output
        **kwargs).stdout
      File "/Users/snorre/.pyenv/versions/3.6.7/lib/python3.6/subprocess.py", line 418, in run
        output=stdout, stderr=stderr)
    subprocess.CalledProcessError: Command '['pkg-config', '--libs', '--cflags', 'yajl']' returned non-zero exit status 1.
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

You have some requirements/dependencies, but shall I follow them to install or what is the way to go?

**Comment by vongohren** [comment]
I guess the wheels of yajl is not part of this. Everything worked fine when I added the yajl to my Mac through brew install. Guess I might have to add it to the container as well?

Will close this when Im doen with the task

**Comment by AMDmi3** [comment]
The dependency list is there for reason, so yes, you need `yajl`. Not sure what container you refer to though.

**Comment by vongohren** [comment]
@AMDmi3 I will closet this now. Part of the issue was because of my knowledge with python and how the ecosystem works, and part because of some lacking beginner documentation.
Fex a pointer to installation docs for dependencies.

Container is my docker container, and now it has been added to that as well, so everything runs smoothly now. 

With more experience of the python ecosystem I was able to tackle this, and I guess it is not your job to  learn beginners how to utalize the ecosystem, unless you want your lib to be wider used than by only knowlegdable pythoneers.

Thanks for the reply, and thanks for a good lib
